src/index.ts

```diff
@@ -4,3 +4,4 @@ export { auctioneerMetadata };
 import deploymentAddresses from '../deployments/addresses.json';
 export { deploymentAddresses };
 export * from './proxy';
+export { DAPPS } from './generated/dapps';
```

Comment:
For example `data-feed-reader-example` needs to import this to prevent proxy deployments with unknown dApp aliases